docs/reference/commandline/logs.md

```diff
@@ -30,8 +30,9 @@ The `docker logs` command batch-retrieves logs present at the time of execution.
 
 > **Note**
 >
-> This command is only functional for containers that are started with the
-> `json-file` or `journald` logging driver.
+> This command is only functional for containers that are started with the `local`,
+> `json-file` or `journald` logging drivers, unless
+> [dual logging](https://docs.docker.com/config/containers/logging/dual-logging/) is enabled.
```

Comment:
Given that dual logging is now automatically used if needed, perhaps the better options would be to remove this note as a whole (effectively it now "just works", so removing the block reduces noise)